### DISCUSSION TITLE
Fixing indentation (=) in go

### DISCUSSION THREAD

**Discussion by realh** [VSCodeVim/Vim#8310] [Q&A]
When editing go code with the official vscode-go extension, vim's = operator, which is supposed to fix the indentation of the selection/range, fails to do anything. Is this a bug in either the go or vim extension? Is there a function I could rebind = to? At a stretch I could rebind it to just save the buffer, because in go this automatically reformats the entire file, and it doesn't really matter to me if it does that instead of just the current line/selection. But I've got barely any knowledge/experience of reconfiguring vscode-vim key bindings. How do I get it to work properly across all the vim contexts/modes and only for go code?

**Comment by max-sixty** [comment]
Does running "Format Selection" work? If not, this is an issue with VS Code or the Go extension, rather than the VIM extension...

**Reply by realh** [reply]
I can't find "Format Selection" anywhere, only "Format Document" in the command palette. That does work. There's a "Reindent Selected Lines", which sort of works, but doesn't seem to understand go syntax, because it gets the indentation all wrong.

**Reply by max-sixty** [reply]
Then this is probably not a VIM issue — VIM just calls those commands
